src/Microsoft.AspNet.Razor.Runtime/TagHelpers/ITagHelper.cs

```diff
@@ -17,6 +17,16 @@ public interface ITagHelper
         int Order { get; }
```

Comment:
"executed" is now vague. Mention `Init()` and `Process()` / `ProcessAsync()` methods are both invoked in given order and `Init()` methods are all done prior to `Process()` / `ProcessAsync()` methods.